pkg/apiserver/server.go

```diff
@@ -21,6 +22,10 @@ func Start() {
 		log.Println("Failed to run migrations", err)
 	}
 
+	if err := informers.Start(); err != nil {
+		log.Println("Failed to start informers", err)
```

Comment:
no panic here?

Reply:
what about if there's no velero?

Reply:
yeah... i guess it's good enough for now